.circleci/config.yml

```diff
@@ -0,0 +1,128 @@
+      ls workspace/cdat
+      # following will install miniconda3 under $WORKDIR/miniconda/bin
+      python workspace/cdat/scripts/install_miniconda.py -w $WORKDIR -p 'py3'
+      
+  - &create_conda_env
+    name: create_conda_env
+    command: |
+       export PATH=$WORKDIR/miniconda/bin:$PATH
+       conda config --set always_yes yes --set changeps1 no
+       conda update -y -q conda
+       conda config --set anaconda_upload no
+       conda create -n py2 -c cdat/label/nightly -c conda-forge -c cdat vcs testsrunner "python<3"
+       #conda create -n py3 -c cdat/label/nightly -c conda-forge -c cdat vcs testsrunner "python>3"
+
+
+  - &setup_vcsjs
+    name: setup_vcsjs
+    command: |
+       export PATH=$WORKDIR/miniconda/bin:$PATH
+       export UVCDAT_ANONYMOUS_LOG=False
+       set +e
+       #source activate py3
+       source activate py2
+       set -e
+       rm -rf build
+
+  - &run_vcsjs_tests
+    name: run_vcsjs_tests
+    command: |
+       export PATH=$WORKDIR/miniconda/bin:$PATH
+       export UVCDAT_ANONYMOUS_LOG=False
+       set +e
+       source activate py2
+       set -e
+       #python run_tests.py -H -v2 
+       export PYTHONPATH=.
+       vtkpython scripts/vcs-test-server.py &
+       npm run karma:travis:wait
+       RESULT=$?
+       #echo "**** py2 test result: "${RESULT}
+       #set +e
+       #source activate py3
+       #set -e
+       #python run_tests.py -H -v2 
+       #PY3_RESULT=$?
+       #echo "**** py3 test result: "${PY3_RESULT}
+       #RESULT=$(( $RESULT + $PY3_RESULT))
+       exit $RESULT
+
+  - &conda_upload
+    name: conda_upload
+    command: |
+       if [[ $CIRCLE_BRANCH != 'master' ]]; then
+          exit 0
+       fi
+       export PATH=${HOME}/project/$WORKDIR/miniconda/bin:$PATH
+       conda install conda-build anaconda-client
+       conda config --set anaconda_upload no
+       export CONDA_BLD_PATH=`pwd`/conda_build
+       git clone git://github.com/cdat/conda-recipes
+       cd conda-recipes
+       export PKG_NAME=vcs-js
+       export USER=cdat
+       export VERSION=8.0
+       export LABEL=nightly
+       python ./prep_for_build.py -l $VERSION
+       conda build $PKG_NAME -c cdat/label/nightly -c conda-forge -c cdat --python=2.7
+       #conda build $PKG_NAME -c cdat/label/nightly -c conda-forge -c cdat --python=3.6
+       anaconda -t $CONDA_UPLOAD_TOKEN upload -u $USER -l $LABEL $CONDA_BLD_PATH/$OS/$PKG_NAME-$VERSION.`date +%Y*`0.tar.bz2 --force
+
+jobs:
+  macos_vcsjs:
+    macos:
+      xcode: "9.2.0"
+    environment:
+      WORKDIR: "workspace/test_macos_vcsjs"
+      OS: "osx-64"
+    steps:
+      - checkout
+      - run: *setup_miniconda
+      - run: *create_conda_env
+      - run: *setup_vcsjs
+      - run: *run_vcsjs_tests
+      - run: *conda_upload
+      - store_artifacts:
+          path: tests_html
+          destination: tests_html
+
+  linux_vcsjs:
+    machine:
+      image: circleci/classic:latest
```

Comment:
It might make sense to try one of the images that adds `-browsers` at the end, as hinted at here:

https://discuss.circleci.com/t/installing-chrome-inside-of-your-docker-container/9067/4

I think we might still need to run `xvfb` first, as that post describes a little further up.